apps/bfd-data-fda/src/main/java/gov/cms/bfd/data/fda/lookup/FdaDrugCodeDisplayLookup.java

```diff
@@ -141,23 +85,25 @@ public String retrieveFDADrugCodeDisplay(Optional<String> claimDrugCode) {
   }
 
   /**
-   * Gets the processed FDA data and checks whether the fake drug code needs to be inserted into the
-   * map.
+   * Constructs a {@link FdaDrugCodeDisplayLookup} for testing purposes only.
```

Comment:
Is this comment accurate that it's for testing purposes only?

Reply:
that constructor is currently used for testing only

Reply:
Oh! That's okay. I just expected any comment about testing to be in the test class but I see what you mean now